doc/contributing/collaborator-guide.md

```diff
@@ -392,13 +392,21 @@ metadata. Raise a pull request like any other change.
 
 Treat commits that introduce new core modules with extra care.
 
-Check if the module's name conflicts with an existing ecosystem module. If it
-does, choose a different name unless the module owner has agreed in writing to
-transfer it.
+New modules must only be added with the `node:` prefix.
 
-If the new module name is free, register a placeholder in the module registry as
-soon as possible. Link to the pull request that introduces the new core module
-in the placeholder's `README`.
+When adding promises to an existing API, add `/promises`
+(`inspector/promises`, etc.). Apply the `semver-major` label to the addition.
+
+If the new module name is free in npm, register
+a placeholder in the module registry as soon as possible. Link to the pull
+request that introduces the new core module in the placeholder's `README`.
+
+If the module name is not free and the module is
+not widely used, contact the owner to see if they would be willing to transfer
+it to the project.
+
```

Comment:
```suggestion

If the module name is not free and the module is
widely used, consider choosing another name to avoid
user confusion and security risks.

```

Reply:
What's the difference? As long as the owner is willing to transfer ownership to the project, it shouldn't matter.

Reply:
Maybe it should be reworded, but my point is that we should try to avoid using the same name as a widely used module, not take it over.

Reply:
I don't know if it's necessary to have it in this document, I feel like the module name would almost certainly be discussed in the PR trying to add the new module. Also, if we're trying to vendor-in an existing widely used module, using the same name seems like an obvious choice (e.g. `unidici` -> `node:unidici`).